tools/challenge-parser/parser/plugins/add-fill-in-the-blank.js

```diff
@@ -40,19 +42,86 @@ function plugin() {
     if (fillInTheBlankNodes.length > 0) {
       const fillInTheBlankTree = root(fillInTheBlankNodes);
 
-      validateBlanksCount(fillInTheBlankTree);
+      validateBlanksSectionCount(fillInTheBlankTree);
 
       const sentenceNodes = getSection(fillInTheBlankTree, '--sentence--');
       const blanksNodes = getSection(fillInTheBlankTree, '--blanks--');
 
-      const fillInTheBlank = getfillInTheBlank(sentenceNodes, blanksNodes);
-
-      file.data.fillInTheBlank = fillInTheBlank;
+      const lang = file.data.lang;
+      const inputType = file.data.inputType;
+      const toHtml = createMdastToHtml(lang);
+
+      file.data.fillInTheBlank = getFillInTheBlank(sentenceNodes, blanksNodes);
+
+      function getFillInTheBlank(sentenceNodes, blanksNodes) {
```

Comment:
I moved the function inside `transformer()` so it can use `toHtml`, and slightly renamed it (`getfillInTheBlank` to `getFillInTheBlank`).

The function body is the same, except...